src/Transformers/TransformerFactory.php

```diff
@@ -98,7 +97,7 @@ protected function injectDependencies(TransformerInterface $transformer): Transf
     /**
      * Create a new transformer item for a model.
      *
-     * @param Model $model
+     * @param object $model
      * @return TransformerItem
      */
     public function createTransformerItem($model): TransformerItem
```

Comment:
Can you add `object` as a typehint?

Reply:
The `object` typehint is a php 7.2 feature. Right now this package requires a minimum version of php 7.1. I could upgrade this number, but I don't know if some project will break if I do that.

Reply:
Okay, leave it for now. :)